content/hi/service-proxy.md

```diff
@@ -0,0 +1,28 @@
+---
+शीर्षक: सर्विस प्रॉक्सी (Service Proxy)
+status: Completed
+category: टैकनोलजी
+tags: ["नेटवर्किंग"]
+---
+
+सर्विस प्रॉक्सी किसी दिए गए [सर्विस](/service/) से या उससे आने वाले ट्रैफ़िक को रोकता है,
+उसमे फिर कुछ नियम लागू करता है, फिर उस ट्रैफ़िक को किसी अन्य सर्विस पर आगे भेजता है
+यह अनिवार्य रूप से एक "मध्यस्थ" के रूप में कार्य करता है जो नेटवर्क ट्रैफ़िक के बारे में जानकारी एकत्र करता है और उस पर नियम लागू करता है।
+
+## यह समस्या का समाधान करता है
+
+सर्विस से सर्विस संचार (उर्फ नेटवर्क ट्रैफ़िक) का ट्रैक रखने के लिए और
+संभावित रूप से इसे परिवर्तित या पुन: निर्देशित करने के लिए, हमें डेटा एकत्र करने की आवश्यकता होती है।
+परंपरागत रूप से, डेटा संग्रह और नेटवर्क ट्रैफ़िक प्रबंधन को सक्षम करने वाला कोड प्रत्येक एप्लिकेशन के भीतर एम्बेड किया गया था।
+
+## यह कैसे मदद करता है
+
+एक सर्विस प्रॉक्सी हमें इस कार्यक्षमता को "एक्सटर्नालिजे" करने की अनुमति देती है।
+अब इसे ऐप्स के भीतर रहने की आवश्यकता नहीं होगी।
+इसके बजाय, अब इसे प्लेटफ़ॉर्म परत (जहां आपके ऐप्स चलते हैं) में एम्बेड किया जायेगा।
+
+सर्विस के बीच द्वारपाल के रूप में कार्य करते हुए, प्रॉक्सी यह जानकारी प्रदान करती है कि किस प्रकार का संचार हो रहा है।
+अपनी अंतर्दृष्टि के आधार पर, वे यह निर्धारित करती है कि किसी विशेष अनुरोध को कहां भेजना है या इसे पूरी तरह से अस्वीकार करना है।
```

Comment:
Replace the word 'अंतर्दृष्टि' with something else.

Reply:
update the 'अंतर्दृष्टि' with the 'निरीक्षण'